### PR TITLE
DeleteCalendarByIdAsync in API service added.

### PR DESCRIPTION
**Pull request by tajwal**

### CODE REVIEW

File: OtripleS.Web.Api.Tests.Unit/Services/StudentExams/StudentExamServiceTests.cs [hunk 1]
```diff
@@ -41,9 +41,6 @@ public StudentExamServiceTests()
         private static DateTimeOffset GetRandomDateTime() =>
             new DateTimeRange(earliestDate: new DateTime()).GetValue();
 
-        private IQueryable<StudentExam> CreateRandomStudentGuardians() =>
-           CreateStudentExamFiller(DateTimeOffset.UtcNow).Create(GetRandomNumber()).AsQueryable();
-
         private StudentExam CreateRandomStudentExam(DateTimeOffset dateTime) =>
```
Comment (hassanhabib) [inline OtripleS.Web.Api.Tests.Unit/Services/StudentExams/StudentExamServiceTests.cs:44]: Why are we deleting this?

Comment (tajwal) [reply OtripleS.Web.Api.Tests.Unit/Services/StudentExams/StudentExamServiceTests.cs:44]: StudentExamServiceTests has CreateRandomStudentExam and this method is irrelevant  CreateRandomStudentGuardians which was causing build error in Azure pipelines.

**Comment by hassanhabib** [comment]
@tajwal please address conflicts

**Comment by tajwal** [comment]
> @tajwal please address conflicts


Mr. @hassanhabib conflicts.

**Pull request merged**